root/static/scripts/edit/MB/Control/ArtistCredit.js

```diff
@@ -76,7 +76,9 @@ MB.Control.ArtistCreditName = aclass(MB.entity.ArtistCreditName, {
 
         } else if (/^&$/.test(join)) {
             this.joinPhrase(" & ");
-        }
+        }else if(/^;$/.test(join) ){
```

Comment:
s/could/should/ ;)

We generally follow [PEP8](https://www.python.org/dev/peps/pep-0008/) with regards to writing Python code. This is the standard used for the Python project itself and far most Python projects adopt most or all of this guideline as well, so it is useful to be familiar with it and get used to writing in the style it dictates if you plan on contributing to projects using Python code (within MetaBrainz or elsewhere). :)